bitsandbytes/__main__.py

```diff
@@ -46,26 +32,31 @@ def generate_bug_report_information():
     print('')
 
     if 'CONDA_PREFIX' in os.environ:
-        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*so')
+        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*')
         print_header("ANACONDA CUDA PATHS")
         print(paths)
         print('')
     if isdir('/usr/local/'):
-        paths = find_file_recursive('/usr/local', '*cuda*so')
+        paths = find_file_recursive('/usr/local', '*cuda*')
         print_header("/usr/local CUDA PATHS")
         print(paths)
         print('')
+    if 'CUDA_PATH' in os.environ and isdir(os.environ['CUDA_PATH']):
+        paths = find_file_recursive(os.environ['CUDA_PATH'], '*cuda*')
+        print_header("CUDA PATHS")
+        print(paths)
+        print('')
 
     if isdir(os.getcwd()):
-        paths = find_file_recursive(os.getcwd(), '*cuda*so')
+        paths = find_file_recursive(os.getcwd(), '*cuda*')
         print_header("WORKING DIRECTORY CUDA PATHS")
         print(paths)
         print('')
 
     print_header("LD_LIBRARY CUDA PATHS")
     if 'LD_LIBRARY_PATH' in os.environ:
         lib_path = os.environ['LD_LIBRARY_PATH'].strip()
-        for path in set(lib_path.split(':')):
+        for path in set(lib_path.split(':' if not os.sep == "\\" else ";")):
```

Comment:
I think this is might be incorrect. What is the `os.sep` for linux? I think `:` is a custom separator that is just for linux env variables. Can you please check this?

Reply:
now fixed to use `os.pathsep` https://docs.python.org/3/library/os.html#os.pathsep